### ISSUE TITLE
Licence

### ISSUE THREAD

**Issue by norwindijkman** [stephane-monnot/resume#1]
Hello Stephane, Your resume is build very nicely and clean i my opinion. If that it possible I would consider using it as a starting-point template for a project. However I don't see any licence. Could you please provide some more clarity regarding the intentions of sharing this code. Can I copy some parts if I provide credits in the source?

**Comment by stephane-monnot** [comment]
Yes you can. I will add Licence file soon. Sorry for the inconvenience.
